.devcontainer/devcontainer.json

```diff
@@ -5,8 +5,9 @@
 	// Or use a Dockerfile or Docker Compose file. More info: https://containers.dev/guide/dockerfile
 	"image": "mcr.microsoft.com/devcontainers/python:1-3.11-bookworm",
 	"features": {
-		"ghcr.io/devcontainers/features/docker-in-docker:2": {}
-		// "ghcr.io/devcontainers/features/node:1": {},
+		"ghcr.io/devcontainers/features/docker-in-docker:2": {},
+		"ghcr.io/devcontainers/features/node:1": {},
+		// "ghcr.io/devcontainers-contrib/features/typescript:2": {}
```

Comment:
Is typescript required in the devcontainer? Is this somehow related to the failing CI/CD?